integration-libs/cdc/manage-users/disable-edit/cdc-user-details.component.ts

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Rename the enclosing folder from `disable-edit` to `user-details`

Reply:
done